runhouse/servers/obj_store.py

```diff
@@ -333,10 +333,15 @@ def keys(self) -> List[Any]:
     # KV Store: Put
     ##############################################
     @staticmethod
-    def put_for_env_servlet_name(env_servlet_name: str, key: Any, value: Any):
-        logger.info(f"Putting {key} and {value} into servlet {env_servlet_name}")
+    def put_for_env_servlet_name(
+        env_servlet_name: str, key: Any, data: Any, serialization: Optional[str] = None
+    ):
         return ObjStore.call_actor_method(
-            ObjStore.get_env_servlet(env_servlet_name), "put_local", key, value
+            ObjStore.get_env_servlet(env_servlet_name),
+            "put_local",
+            key,
+            data=data,
+            serialization=serialization,
```

Comment:
Same comment as on earlier PR - if we're already in the env, we should just call self.put_local rather than jump through Ray and the EnvServlet into this very object store, no?